server/src/main/resources/rules/rules.js

```diff
@@ -2351,9 +2423,18 @@ var Autobind = {
                 var provided_size = this.get_provided_products().length;
                 for (var i = this.pools.length - 1; i >= 0; i--) {
                     temp = this.pools[i];
+
+                    var roleMatch = role == temp.getProductAttribute("role");
+                    var poolAddons = temp.getProductAttribute("addons") != null ?
+                        temp.getProductAttribute("addons").split(","):[];
```

Comment:
I'm not sure if JS's `split` takes a regex, but if it does we'll want to account for spaces here too.